.travis.yml

```diff
@@ -3,5 +3,17 @@ sudo: false
 go:
   - 1.5
   - 1.6
+  - 1.7
+
+before_install:
+  - go get github.com/mattn/goveralls
```

Comment:
It's required to push the coverage profile from travis to coveralls. It's the easiest way to track the test coverage.